docs/docsite/rst/user_guide/playbooks_tags.rst

```diff
@@ -1,16 +1,18 @@
 Tags
 ====
 
-If you have a large playbook it may become useful to be able to run a specific part of the configuration without running the whole playbook.
+If you have a large playbook, it may become useful to be able to run only
+a specific part of it rather than running *everything* in the playbook.
+Ansible supports a "tags:" attribute for this reason.
 
-Both plays and tasks support a "tags:" attribute for this reason.
-You can **ONLY** filter tasks based on tags from the command line with ``--tags`` or ``--skip-tags``.
-Adding "tags:" in any part of a play (including roles) adds those tags to the contained tasks.
+You can **ONLY** filter tasks based on tags from the command line with the 
+``--tags`` or ``--skip-tags`` options.
```

Comment:
this is not really true anymore, you can do so via config also

Reply:
Since this is the documentation itself, can you explain how that's done, (or where else it's documented), so I can explain it here?

Reply:
...wait, maybe I found what you're talking about. Are you referring to the https://docs.ansible.com/ansible/devel/reference_appendices/config.html?highlight=ansible_skip_tags#tags-run and https://docs.ansible.com/ansible/devel/reference_appendices/config.html?highlight=ansible_skip_tags#tags-skip configuration parameters?

Reply:
@bcoca Added remarks about config skip/run, as requested.